client/components/LearnerViewDirective.js

```diff
@@ -568,6 +586,33 @@ tie.directive('learnerView', [function() {
           }
         };
 
+        $scope.changeQuestionSet = function() {
+          questionSetId = '';
+          switch ($scope.currentQuestionSetId) {
```

Comment:
Cleaner to have, somewhere:

    var ALLOWED_QUESTION_SET_IDS = [...];

and then do

    if (ALLOWED_QUESTION_SET_IDS.indexOf(questionSetId) === -1) {
      return;
    }

    ...